.github/workflows/ci.yml

```diff
@@ -5,7 +5,19 @@ on:
   push:
     branches:
       - main
+    paths:
+      - '**'
+      - '!**/README*'
+      - '!**/CONTRIBUTING*'
+      - '!./.github/**'
+      - './.github/workflows/ci.yml'
```

Comment:
Changes to CI and workflows would seem to be an excellent thing to trigger a CI workflow.